src/main/resources/api-specs/cart-openapi.yaml

```diff
@@ -0,0 +1,192 @@
+openapi: "3.0.3"
+
+info:
```

Comment:
we need to write in a common style as in Zufar's example in product-openapi.yaml

info:
  title: "Online-Store API"
  version: "1.0.0"

Reply:
fixed